### ISSUE TITLE
[Bug]  Connection refused on android

### ISSUE THREAD

**Issue by henriSedjame** [fzyzcjy/flutter_rust_bridge#473]
### Describe the bug

Using:

 * flutter_rust_bridge = "1.30.0"  on cargo.toml
 *  flutter_rust_bridge: ^1.31.0 on pubspec.yaml
 
I meet a connection error while trying to run my android app.

With IOS no problem !!!

This is the error:

<img width="1572" alt="Capture d’écran 2022-05-24 à 19 37 25" src="https://user-images.githubusercontent.com/32457325/170100452-c4820d06-e089-473e-be4b-ef9979484d95.png">

This the generated JniLibs dir :

<img width="468" alt="Capture d’écran 2022-05-24 à 19 37 51" src="https://user-images.githubusercontent.com/32457325/170100690-c01f0f9b-87ea-43fe-9b14-4d69affeb505.png">

And this how I integrate the generated lib into my flutter code:

<img width="748" alt="Capture d’écran 2022-05-24 à 19 38 37" src="https://user-images.githubusercontent.com/32457325/170100813-914b5d59-7640-4edf-b11a-a5e0a21e71e0.png">

Hope you will be able to help me :)

### Codegen logs with `RUST_LOG=debug` env variable

```shell
2022-05-24T17:44:58Z DEBUG lib_flutter_rust_bridge_codegen::commands] execute format_dart path=/Users/henrijoelsedjame/development/TESTS/FLUTTER/todo_app/lib/bridge_generated.dart line_length=80
[2022-05-24T17:44:58Z DEBUG lib_flutter_rust_bridge_codegen::commands] execute command: bin=sh args=["-c", "dart format /Users/henrijoelsedjame/development/TESTS/FLUTTER/todo_app/lib/bridge_generated.dart --line-length 80"] current_dir=None cmd="sh" "-c" "dart format /Users/henrijoelsedjame/development/TESTS/FLUTTER/todo_app/lib/bridge_generated.dart --line-length 80"
[2022-05-24T17:44:59Z DEBUG lib_flutter_rust_bridge_codegen::commands] command="sh" "-c" "dart format /Users/henrijoelsedjame/development/TESTS/FLUTTER/todo_app/lib/bridge_generated.dart --line-length 80" stdout=Formatted /Users/henrijoelsedjame/development/TESTS/FLUTTER/todo_app/lib/bridge_generated.dart
    Formatted 1 file (1 changed) in 0.47 seconds.
     stderr=
[2022-05-24T17:44:59Z INFO  lib_flutter_rust_bridge_codegen] Success!
[2022-05-24T17:44:59Z INFO  flutter_rust_bridge_codegen] Now go and use it :)
```

### OS

Android

### Version of `flutter_rust_bridge_codegen`

1.30.0

**Comment by fzyzcjy** [comment]
That seems a bug of your rust code instead of frb. Check network permission and your rust code.

**Comment by henriSedjame** [comment]
You're right, the problem come from the connection to postgresql database.
I don't know why, but it works fine with IOS.
Adding permissions to my android manifest file do not change anything.
I'm still investigating.

thank you for taking time to respond ;)

**Comment by fzyzcjy** [comment]
You are welcome :)
